File: backend/src/main/java/edu/cit/mabini/meditrack/dto/DoctorDto.java
package edu.cit.mabini.meditrack.dto;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class DoctorDto {
    private Long id;
    private String fullName;
    private String email;
    private String phoneNumber;
    private String specialization;
    private String licenseNumber;
    private String role;
    private LocalDateTime createdAt;
}

File: mobile/app/src/main/java/edu/cit/mabini/meditrack/feature/doctors/data/model/DoctorDto.kt
package edu.cit.mabini.meditrack.feature.doctors.data.model

data class DoctorDto(
    val id: Long,
    val fullName: String,
    val email: String? = null,
    val phoneNumber: String? = null,
    val specialization: String? = null,
    val licenseNumber: String? = null,
    val role: String? = null
)
